website/static/js/filepage/index.js

```diff
@@ -363,6 +364,37 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+        function goToRevisions(e){
+            var editable = self.editor && self.editor.selected;
```

Comment:
Rather than doing all this work to manipulate the DOM and modify the controller state, could you just link to `?show=revision`?

We modify the user's query params in other parts of the site. `website/statis/js/search.js` should have examples of this.

This will allow the user to use the "Back" button to go back to the default file view. It will also allow the user to copy and paste the URL for revisions.